### ISSUE TITLE
Unexpected error compiling the bridge in 1.3.3

### ISSUE THREAD

**Issue by marcin-rzeznicki** [scalacenter/bloop#1051]
After upgrading to `1.3.3`, `bloop compile x` fails with:
```
Attempting to fetch ch.epfl.scala:compiler-bridge_2.12:1.3.0-M4+26-045d2c8c.
[W]     module not found: ch.epfl.scala#compiler-bridge_2.12;1.3.0-M4+26-045d2c8c
[W] ==== local: tried
[W]   /home/rzeznik/.ivy2/local/ch.epfl.scala/compiler-bridge_2.12//1.3.0-M4+26-045d2c8c/ivys/ivy.xml
[W] ==== public: tried
[W]   https://repo1.maven.org/maven2/ch/epfl/scala/compiler-bridge_2.12/1.3.0-M4+26-045d2c8c/compiler-bridge_2.12-1.3.0-M4+26-045d2c8c.pom
[W] ==== sonatype-staging: tried
[W]   https://oss.sonatype.org/content/repositories/staging/ch/epfl/scala/compiler-bridge_2.12/1.3.0-M4+26-045d2c8c/compiler-bridge_2.12-1.3.0-M4+26-045d2c8c.pom
[W]     ::::::::::::::::::::::::::::::::::::::::::::::
[W]     ::          UNRESOLVED DEPENDENCIES         ::
[W]     ::::::::::::::::::::::::::::::::::::::::::::::
[W]     :: ch.epfl.scala#compiler-bridge_2.12;1.3.0-M4+26-045d2c8c: not found
[W]     ::::::::::::::::::::::::::::::::::::::::::::::
[W] 
Compiled shared (5ms)
sbt.internal.inc.InvalidComponent: The compiler bridge sources sbt.internal.librarymanagement.IvySbt$Module@5441662e could not be retrieved.

        Note: Unresolved dependencies path:
                ch.epfl.scala:compiler-bridge_2.12:1.3.0-M4+26-045d2c8c
                  +- org.scala-sbt.temp:temp-module-98ec89c315ed408f615fff2c4774286d17323b01:1.3.0-M4+26-045d2c8c
```

_jar_ it complains is published to [MC](https://mvnrepository.com/artifact/ch.epfl.scala/compiler-bridge_2.12/1.3.0-M4+26-045d2c8c). I can locate it in coursier cache on my machine.

**Comment by jvican** [comment]
@marcin-rzeznicki Thanks for reporting! This issue is not Arch-specific and there's already a fix for it. I'll release 1.3.4 on Monday early in the morning with that fix included, in the meanwhile you can use 1.3.2 as a workaround.

**Comment by marcin-rzeznicki** [comment]
@jvican Thank you! I thought that it might be caused by `pacaur` install (BTW, I use Arch Linux :smiley: ). Keep up the good work!
